### PR TITLE
feat: add support to include annotations to kratos configmap

### PR DESCRIPTION
**Pull request by cpwc**
## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got green light (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: helm/charts/kratos/templates/configmap-config.yaml [hunk 1]
```diff
@@ -6,6 +6,10 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
 {{ include "kratos.labels" . | indent 4 }}
+  annotations:
+    {{- with .Values.configmap.annotations }}
+      {{- toYaml . | indent 4 }}
```
Comment (Demonsthere) [inline helm/charts/kratos/templates/configmap-config.yaml:11]:
```suggestion
      {{- toYaml . | nindent 4 }}
```

File: helm/charts/kratos/templates/configmap-migrate.yaml [hunk 2]
```diff
@@ -10,6 +10,9 @@ metadata:
     helm.sh/hook-weight: "0"
     helm.sh/hook: "pre-install, pre-upgrade"
     helm.sh/hook-delete-policy: "before-hook-creation"
+    {{- with .Values.configmap.annotations }}
+      {{- toYaml . | indent 4 }}
```
Comment (Demonsthere) [inline helm/charts/kratos/templates/configmap-migrate.yaml:14]:
```suggestion
      {{- toYaml . | nindent 4 }}
```

File: helm/charts/kratos/templates/configmap-templates.yaml [hunk 3]
```diff
@@ -9,6 +9,10 @@ metadata:
   namespace: {{ $root.Release.Namespace }}
   labels:
 {{ include "kratos.labels" $root | indent 4 }}
+  annotations: 
+    {{- with .Values.configmap.annotations }}
+      {{- toYaml . | indent 4 }}
```
Comment (Demonsthere) [inline helm/charts/kratos/templates/configmap-templates.yaml:14]:
```suggestion
      {{- toYaml . | nindent 4 }}
```

**Comment by Demonsthere** [comment]
Thanks for the PR! I have took the liberty to add a small change and merge it 😉

**Comment by cpwc** [comment]
Appreciate it! Thank you so much 🥳

**Pull request merged**